styleguide/components/41-organisms/footer/_footer.scss

```diff
@@ -120,7 +120,7 @@ footer.footer {
   }
 
   .gentinfo-chat-button i {
-    font-size: 1.2rem;
     margin-top: .2rem;
+    font-size: 1.2rem;
```

Comment:
Does this deviate from the 'button-with-icon' style?

Reply:
Yes, it's bigger.

Reply:
It shouldn't be